jekyll/_data/api.yml

```diff
@@ -389,6 +389,20 @@ add_user_ssh:
     This allows them to SSH to the build containers.
   response: '{... the build data ... }'
 
+project_build:
+  url: "/api/v1.1/project/:vcs-type/:username/:project/build"
+  description: "Trigger a build of the specified project, by branch / revision or tag. All workflows found in the build configuration will be run."
+  method: "POST"
+  params:
+    - {name: "branch", description: "The branch to build. Cannot be used with tag parameter."}
+    - {name: "revision", description: "The specific revision to build. If not specified, the HEAD of the branch is used. Cannot be used with tag parameter", note: "Branch and revision can either be used separately or together. If they are used together, the revision is built but the branch is used to display the correct branch on the job page."}
+    - {name: "tag", description: "The git tag to build. Cannot be used with branch and revision parameters."}
+  body: |
```

Comment:
this should be under `response` rather than `body` - in this case the `body` would be an example of using the above parameters, but for the short-term I suspect we can omit `body`.

Reply:
thanks, fixed